src/pages/Projects.jsx

```diff
@@ -1,19 +1,172 @@
+// React Router Imports
+import { Link as ReactRouterLink } from 'react-router-dom';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Chip from '@mui/material/Chip';
+import Divider from '@mui/material/Divider';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
+// Custom Imports
 import Hero from '../components/home/Hero';
-import ProjectBox from '../components/projects/ProjectBox';
-import { Container } from '@mui/material';
+import projectsList from '../components/projects/projectsList';
+
+const renderingLinks = (href, icon) => {
+  return (
+    <Link
+      component={ReactRouterLink}
+      key={href}
+      to={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      sx={linkStyle}
+    >
+      {icon}
+    </Link>
+  );
+};
+
+const boxStyle = (index) => ({
+  mb: { xs: 5, sm: 15 },
+  p: 5,
+  borderRadius: '25px',
+  minWidth: { xs: '95vw', sm: '80vw' },
+  maxWidth: { sm: '85vw' },
+  background:
+    index % 2 === 0
```

Comment:
something you had commented on secondary partners, can we make the gradient go up and down on smaller viewports. If we want to change how the gradient looks after that we can figure out at the meeting but at least it will be consistent for now. 

```js
 background: {
    md:
      index % 2 === 0
        ? 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
        : 'linear-gradient(90deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)',
    xs: 'linear-gradient(0deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
  }
```